nvme.c

```diff
@@ -3075,11 +3084,58 @@ static int create_ns(int argc, char **argv, struct command *cmd, struct plugin *
 		return -EINVAL;
 	}
 
-	err = parse_lba_num_si(dev, "nsze", cfg.nsze_si, cfg.flbas, &cfg.nsze);
+	id = nvme_alloc(sizeof(*id));
+	if (!id)
+		return -ENOMEM;
+
+	err = nvme_identify_ctrl(dev_fd(dev), id);
+	if (err) {
+		if (err < 0) {
+			nvme_show_error("identify-controller: %s", nvme_strerror(errno));
+		} else {
+			fprintf(stderr, "identify controller failed\n");
+			nvme_show_status(err);
+		}
+		return err;
+	}
+
+	align_nsze = align_ncap = 1 << 20; /* Default 1 Mb */
```

Comment:
The value can be initialized by the variable definitions as for example. By the way `1 Mb` should be `1 MiB`. (The lowercase character `b` also should be uppercase charadter `B`. This is also same for the commit summary `1Mib` and description `1Mb`.)
```c
	__u32 nsid:
	__u32 align_nsze = 1 << 20; /* Default 1 MiB */
	__u32 align_ncap = align_nsze;
```